### PR TITLE
Extend HdfsClient interface

### PR DESCRIPTION
**Pull request by losipiuk**

### CODE REVIEW

File: tempto-core/src/main/java/io/prestosql/tempto/hadoop/hdfs/HdfsClient.java [hunk 1]
```diff
@@ -21,6 +21,7 @@
 import java.io.InputStream;
```
Comment (kokosing) [inline tempto-core/src/main/java/io/prestosql/tempto/hadoop/hdfs/HdfsClient.java:21]: Maybe you could add some example test that test this .

**Review by kokosing — APPROVED** [review]

File: tempto-core/src/main/java/io/prestosql/tempto/hadoop/hdfs/HdfsClient.java (hunk 1 above)
Comment (losipiuk) [reply tempto-core/src/main/java/io/prestosql/tempto/hadoop/hdfs/HdfsClient.java:21]: Oh - do we have product tests for tempto?
We have tempto.examples. But are thy run actually?

Comment (kokosing) [reply tempto-core/src/main/java/io/prestosql/tempto/hadoop/hdfs/HdfsClient.java:21]: Yes, they are run. See the travis: https://travis-ci.com/github/prestosql/tempto/builds/17961737

Comment (losipiuk) [reply tempto-core/src/main/java/io/prestosql/tempto/hadoop/hdfs/HdfsClient.java:21]: Added.

**Pull request merged**